### PR TITLE
Snapshot top n StakeEntries for staking rewards

### PR DESCRIPTION
**Pull request by tholonious**

### CODE REVIEW

**Comment by tholonious** [comment]
Current dependencies on/for this PR:
* feature/proof-of-stake
  * **PR #579** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/579" target="_blank">Graphite</a> 
    * **PR #598** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/598" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/598?utm_source=stack-comment).

File: lib/pos_snapshot_entries.go [hunk 1]
```diff
@@ -675,6 +675,268 @@ func DBPutSnapshotValidatorSetTotalStakeAmountNanosWithTxn(
...
+//
+
+type SnapshotStakeMapKey struct {
+	SnapshotAtEpochNumber uint64
+	ValidatorPKID         PKID
+	StakerPKID            PKID
+}
+
+// This is a bare bones in-memory only construct used to capture the ValidatorPKID,
+// StakerPKID, and StakeAmountNanos from a StakeEntry that has been snapshot. We
```
Comment (lazynina) [inline lib/pos_snapshot_entries.go:689]:
```suggestion
// StakerPKID, and StakeAmountNanos from a StakeEntry that has been snapshotted. We
```

**Review by lazynina — APPROVED** [review]
just one nit on the comments

File: lib/pos_snapshot_entries.go (hunk 1 above)
Comment (tholonious) [reply lib/pos_snapshot_entries.go:689]: Will fix in next PR so we can avoid one extra CI run.

**Pull request merged**